### DISCUSSION TITLE
Cannot install wazuh on ubuntu 22.04 via UTM

### DISCUSSION THREAD

**Discussion by alaz-aura** [wazuh/wazuh#25499] [Q&A] [answered]
Hi there, I am using [UTM](https://github.com/utmapp/UTM) to run VMs on a MB Pro. 

I have Ubuntu v22.04 installed and tried running the wazuh install script - found [here](https://github.com/wazuh/wazuh/discussions/25499). Unfortunately wazuh reports the VM as not a 64-bit system, even though ... it is?

<img width="1179" alt="Screenshot 2024-09-02 at 10 38 39 AM" src="https://github.com/user-attachments/assets/eee5ab20-a8fa-47dc-88f8-49c2325c0223">

Keen to get wazuh up & running, however I am blocked by this install hurdle. Open to any fixes, cheers!

**Comment by psjd23** [comment]
@alaz-aura check the output of these commands:

`uname -m`
`file /bin/bash`

Post back the results

**Reply by alaz-aura** [reply]
@psjd23  here you are

<img width="816" alt="Screenshot 2024-09-25 at 9 30 41 PM" src="https://github.com/user-attachments/assets/794f68cb-97c0-4752-9dfc-141d222603e2">

**Comment by joaquinsgi** [comment, marked as answer]
Hi! @alaz-aura
The message from the Wazuh installation assistant stating that the script must be executed on a 64-bit system, actually means that the script must be executed on a system with x86_64/AMD64 architecture. I have created the following issue to modify the message: https://github.com/wazuh/wazuh/issues/25998.

On the other hand, the Wazuh indexer and Wazuh dashboard components for ARM64 systems are not available yet, and therefore, an AMD64 system will be needed to install the central components, but if you want to use it on an ARM64 system, you will have to virtualize an AMD64 system and install the central components using the Wazuh installation assistant script or the step-by-step guide.

**Reply by alaz-aura** [reply]
excellent @joaquinsgi , thank you for that extra bit of information. I will now give the amd64 solution a test and report back. Cheers!
